### DISCUSSION TITLE
What's in a logo?

### DISCUSSION THREAD

**Discussion by Shamaoke** [palkan/action_policy#274] [Q&A]
Hi.

Could you explain, what is shown in the Action policy logo (the thing at the top)? I can't make in out by myself.

![Action policy logo](https://actionpolicy.evilmartians.io/assets/images/logo.svg)

Thanks.

**Comment by palkan** [comment]
It's a bicycle wrench, like [this one](https://www.walmart.com/ip/Multiple-Function-Bicycle-Wrench/1353161013?wmlspartner=wlpa&selectedSellerId=101112699).

The logo combines two ideas:

1) Multi-functionality / universality  (such wrench can be with nuts of different size and shape).

2) It depicts the purpose of the library—to "tighten the screws" (or "закручивать гайки" in Russian—that's the actual inspiration).
